dev/cognitiveservices/data-plane/Language/analyzetext.json

```
@@ -1785,6 +2152,23 @@
         "entities"
       ]
     },
+    "EntityTags": {
```

It is singular, right?
```suggestion
    "EntityTag": {
```